streampark-console/streampark-console-service/src/main/assembly/bin/streampark.sh

```diff
@@ -68,6 +68,20 @@ else
   RESET=""
 fi
 
+JAVA_OPTS="""
+  $vmOption
```

Comment:
Why do you move the position of `JAVA_OPTS`? , this will be `vmOption` variable is empty

Reply:
This problem has been fixed. Please review it.

Reply:
> Why do you move the position of `JAVA_OPTS`? , this will be `vmOption` variable is empty

This problem has been fixed. Please review it.